internal/catalogmetadata/client/client_test.go

```diff
@@ -60,76 +66,92 @@ func TestClient(t *testing.T) {
 										"name": "fake1.v9.9.9"
 									}
 								]
-							}`),
-						},
-					})
+							}`)...)
+
+					return objs, nil, catalogContentMap
+				},
+				wantErr: "bundle \"fake1.v9.9.9\" not found in catalog \"catalog-1\" (package \"fake1\", channel \"channel-with-missing-bundle\")",
```

Comment:
Nit: when the string itself has double quotes, use backticks for the string literal:
```suggestion
				wantErr: `bundle "fake1.v9.9.9" not found in catalog "catalog-1" (package "fake1", channel "channel-with-missing-bundle")`,
```

Reply:
Addressed in https://github.com/operator-framework/operator-controller/pull/411/commits/e0aede3cbae8e0c366026d90a88bdd9dc85d1123